clang/lib/Driver/Driver.cpp

```diff
@@ -5111,11 +5193,13 @@ class OffloadingActionBuilder final {
         // When spv online link is supported by all backends, the fallback
         // device libraries are only needed when current toolchain is using
         // AOT compilation.
-        if (isSPIR) {
+        if (isSPIR || isNVPTX) {
           bool UseJitLink =
-              Args.hasFlag(options::OPT_fsycl_device_lib_jit_link,
-                           options::OPT_fno_sycl_device_lib_jit_link, false);
-          bool UseAOTLink = isSpirvAOT || !UseJitLink;
+              isSPIR ? Args.hasFlag(options::OPT_fsycl_device_lib_jit_link,
+                                    options::OPT_fno_sycl_device_lib_jit_link,
+                                    false)
+                     : false;
```

Comment:
```suggestion
              isSPIR && Args.hasFlag(options::OPT_fsycl_device_lib_jit_link,
                                    options::OPT_fno_sycl_device_lib_jit_link,
                                    false);
```
Very minor, but seems fitting given roughly the same is done below.

Reply:
Thanks, change made